index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+include 'functions/allFunctions.php';
+
+$db = connectToDB('pokemon-collection');
+
+$userPokemon = fetchPokemonData($db);
+
+echo '<h1>My Pokemon</h1>';
```

Comment:
This should be part of the HTML rather than being echoed - you'll also want to add in the standard HTML5 boilerplate code.